lib/open_food_network/orders_and_fulfillments_report.rb

```diff
@@ -47,12 +48,12 @@ def header
     end
 
     def search
-      Reports::LineItems.search_orders(permissions, params)
+      Reports::LineItems.search_orders(@permissions, params)
     end
 
     def table_items
       return [] unless @render_table
-      Reports::LineItems.list(permissions, params)
+      Reports::LineItems.list(@permissions, params)
```

Comment:
you wouldn't need this change if we were using ruby's `attr_reader` which I strongly recommend due to the encapsulation and other improvements it provides over ivars.